tests/js/spec/views/issueList/tagFilter.spec.tsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * @jest-environment jest-environment-jsdom-fifteen
+ */
+
+import {fireEvent, mountWithTheme, waitFor} from 'sentry-test/reactTestingLibrary';
+
+import IssueListTagFilter from 'app/views/issueList/tagFilter';
+
+describe('IssueListTagFilter', function () {
+  // @ts-expect-error
+  const routerContext = TestStubs.routerContext();
+
+  // @ts-expect-error
+  MockApiClient.clearMockResponses();
+
+  const selectMock = jest.fn();
+  const tag = {key: 'browser', name: 'Browser'};
+  const tagValueLoader: IssueListTagFilter['props']['tagValueLoader'] = () =>
+    new Promise(resolve =>
+      resolve([
+        {
+          count: 0,
+          firstSeen: '2018-05-30T11:33:46.535Z',
+          key: 'foo',
+          lastSeen: '2018-05-30T11:33:46.535Z',
+          name: 'foo',
+          value: 'foo',
+          id: 'foo',
+          ip_address: '192.168.1.1',
+          email: 'foo@boy.cat',
+          username: 'foo',
+        },
+        {
+          count: 0,
+          firstSeen: '2018-05-30T11:33:46.535Z',
+          key: 'fooBaar',
+          lastSeen: '2018-05-30T11:33:46.535Z',
+          name: 'fooBaar',
+          value: 'fooBaar',
+          id: 'fooBaar',
+          ip_address: '192.168.1.1',
+          email: 'fooBaar@boy.cat',
+          username: 'ffooBaaroo',
+        },
+      ])
+    );
+
+  it('calls API and renders options when opened', async function () {
+    const {getByLabelText, getByTestId, getByText} = mountWithTheme(
+      <IssueListTagFilter
+        tag={tag}
+        value=""
+        onSelect={selectMock}
+        tagValueLoader={tagValueLoader}
+      />,
+      routerContext
+    );
+
+    // changes dropdown input value
+    const input = getByLabelText(tag.key);
+    fireEvent.change(input, {target: {value: 'foo'}});
+
+    // waits for the loading indicator to disappear
+    const loadingIndicator = getByText('Loading...');
```

Comment:
Should this be using an ellipsis or just `Loading` instead?

Reply:
it should be using ellipsis to match the string, otherwise it won't work

Reply:
This is three dots, where as above it's `Loading\u2026` no?

Reply:
`Loading…` does not match `Loading...`, but I will double-check it

Reply:
I wonder if it does something like https://www.npmjs.com/package/normalize-diacritics to keep the text ANSI, 🤔 strange

Reply:
this was happening because react-select was rendering its default load message, which doesn't use Unicode ellipses. I updated the code and now it works, but a way to ignore the ellipses and just check the load string would be:

`const loadingIndicator = getByText(content => content.startsWith('Loading'));`